docs/inputs.md

```diff
@@ -2565,32 +2280,21 @@ oth; where Size_Class ranges 1-7.
 
 <a id="mobility_mgra"></a>
 
-### MGRAs at Mobility Hubs
-#### `MOBILITYHUBMGRA.CSV`
+### Zone Terminal Time
+#### `ZONE.txt`
```

Comment:
is it zone_term.csv file?

Reply:
Creating another PR because of branch issues to address this.